crates/ra_syntax/src/ast/make.rs

```diff
@@ -136,6 +136,15 @@ pub fn placeholder_pat() -> ast::PlaceholderPat {
     }
 }
 
+pub fn tuple_pat(pats: impl IntoIterator<Item = ast::Pat>) -> ast::TuplePat {
+    let pats_str = pats.into_iter().map(|p| p.syntax().to_string()).join(", ");
```

Comment:
Nit: `p.syntax().to_string()` can be just `p.to_string()` due to the recent improvements in the ast API.

Reply:
Fixed.